tools/pullanalize

```diff
@@ -22,6 +22,7 @@ then
         -Dsonar.github.login=magwas\
         -Dsonar.externalIssuesReportPaths=issuesReport.json\
         -Dsonar.java.pmd.reportPaths=target/pmd_full.xml\
+        -Dsonar.tests=src/test,src/integration-test\
```

Comment:
Isn't it needed in the other sonar run as well?

Reply:
It is, but at first I wanted to try it. I couldn't reproduce it with the same mvn command on my machine.